code/modules/medical/genetics/bioEffects/powers.dm

```diff
@@ -1401,19 +1405,22 @@
 		if (!istype(linked_power,/datum/bioEffect/power/dimension_shift))
 			return 1
 		var/datum/bioEffect/power/dimension_shift/P = linked_power
-
 		if (!istype(owner.loc,/turf/) && !istype(owner.loc,/obj/dummy/spell_invis/))
 			boutput(owner, "<span class='alert'>That won't work here.</span>")
 			return 1
-
 		if (P.processing)
 			return 1
 
 		P.processing = 1
 
 		if (!P.active)
+			if (istype(owner.loc,/obj/dummy/spell_invis/)) // check for if theres a spell_invis object we havent placed (from biomass manipulation)
+				// before this, dimension shift and biomass manipulation resulted in strange behavior, including being sent to nullspace.
+				boutput(owner, "<span class='alert'>That won't work here.</span>")
+				P.processing = 0
+				return 1
```

Comment:
```suggestion
				return TRUE
```
Might be a good idea to change the rest to be boolean T/F instead of 1/0 as well, cleanliness

Reply:
done